packages/p2p/src/components/modal-manager/modals/email-link-verified-modal/email-link-verified-modal.tsx

```diff
@@ -2,31 +2,25 @@ import React from 'react';
 import { Button, Icon, Modal, Text } from '@deriv/components';
 import { formatMoney } from '@deriv/shared';
 import { Localize } from 'Components/i18next';
-import { useStores } from 'Stores';
 import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
-import { setDecimalPlaces, removeTrailingZeros, roundOffDecimal } from 'Utils/format-value';
+import { useStores } from 'Stores';
+import { removeTrailingZeros, roundOffDecimal, setDecimalPlaces } from 'Utils/format-value';
 
 const EmailLinkVerifiedModal = () => {
     const { hideModal, is_modal_open } = useModalManagerContext();
     const { order_store } = useStores();
     const { amount_display, is_buy_order_for_user, local_currency, rate } = order_store.order_information || {};
     const amount = removeTrailingZeros(
-        formatMoney(local_currency, amount_display * roundOffDecimal(rate, setDecimalPlaces(rate, 6)), true)
+        formatMoney(local_currency, amount_display * Number(roundOffDecimal(rate, setDecimalPlaces(rate, 6))), true)
```

Comment:
do we need `Number` here? isn't `roundOffDecimal` already returning a number?

Reply:
it returns 0 or string.  there is a toFixed in it.

Reply:
but i see that amount in formatMoney accepts number or string